cmake/iwyu.cmake

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+if(NOT WIN32)
+  include(ProcessorCount)
+  ProcessorCount(CPU_COUNT)
+
+  find_package(Python3 COMPONENTS Interpreter)
+  find_program(iwyu_tool_py NAMES iwyu_tool)
+
+  if (iwyu_tool_py AND TARGET Python3::Interpreter)
+    add_custom_command(OUTPUT iwyu_commands.json
```

Comment:
Can you add a very brief comment about what you are in each of the two custom commands?

Reply:
Yes. good idea.

Reply:
Done